src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -369,6 +369,7 @@ export default {
 			const genderOptions = [
 				{ name: 'Women', key: 'female', __typename: 'GenderEnum' },
 				{ name: 'Men', key: 'male', __typename: 'GenderEnum' },
+				{ name: 'Nonbinary', key: 'nonbinary', __typename: 'GenderEnum' },
```

Comment:
Do we need this change? I'm not sure if this is used anywhere.

@mcstover Is this filter used anywhere?

Reply:
this is in a different context. It "should" be fine to included when this goes live but @michelleinez may want to verify or possibly pull this addition out and update as a separate task.

Reply:
Thanks for the context @mcstover. Maybe be good to revert this file for now and let the other team know.